### ISSUE TITLE
When running parallelDG autocorrelation doesn't plot property 

### ISSUE THREAD

**Issue by melmasri** [felixleopoldo/benchpress#29]
You get a flat picture for size/score correlational when running 
`snakemake --configfile config/parallelDG.json --cores all --use-singularity`

The issues seems to come from this line of code [here](https://github.com/felixleopoldo/benchpress/blob/64e7ba4706b2174107322fae571d1d34c8528349/workflow/scripts/evaluation/plot_autocorr_from_traj.py#L108). the `ffill` routine fills forward, but when the first index in you data is not 0, the re-indixing of this line 
```
        df2 = df2.reindex(newindex).reset_index().reindex(
            columns=df2.columns).fillna(method="ffill")
```
Creates index 0 with NaN.

Here is an example 
```
>>> df2
       size
index      
2         1
4         3
6         6
7        10
8        15
...     ...
99978   197
99980   196
99988   195
99996   194
99998   195

[21120 rows x 1 columns]
>>> df2.reindex(newindex).reset_index()
       index   size
0          0    NaN
1          1    NaN
2          2    1.0
3          3    NaN
4          4    3.0
...      ...    ...
99993  99993    NaN
99994  99994    NaN
99995  99995    NaN
99996  99996  194.0
99997  99997    NaN

[99998 rows x 2 columns]
>>> 99997  99997    NaN

[99998 rows x 2 columns]
>>>   C-c C-c
```

**Comment by melmasri** [comment]
fixed in the above PR #30

**Comment by melmasri** [comment]
merged PR #30
